### PR TITLE
fix posix socket return value for connect

### PR DESCRIPTION
**Pull request by FernetMenta**
I don't know how you do the debian version bump change log thing.

### CODE REVIEW

File: src/posix/os-socket.h [hunk 1]
```diff
@@ -320,8 +320,11 @@ namespace PLATFORM
         else if (iPollResult == -1)
           *iError = errno;
 
-        socklen_t errlen = sizeof(int);
-        getsockopt(socket, SOL_SOCKET, SO_ERROR, (void *)iError, &errlen);
+        if (iError == 0)
```
Comment (opdenkamp) [inline src/posix/os-socket.h]: looks like you're missing some * here? `error` is a pointer.

Comment (opdenkamp) [inline src/posix/os-socket.h]: will fix it up

**Comment by opdenkamp** [comment]
https://github.com/Pulse-Eight/platform/commit/079eded737175fb5fc244c7ad421e1ddc9758c52

File: src/posix/os-socket.h (hunk 1 above)
Comment (FernetMenta) [inline src/posix/os-socket.h]: thanks

**Comment by opdenkamp** [comment]
@wsnipex packages have been uploaded to https://launchpad.net/~pulse-eight/+archive/ubuntu/libcec and are building now

**Comment by FernetMenta** [comment]
@opdenkamp iirc @wsnipex is on holiday. what needs to be done to get this into our build system?

**Comment by opdenkamp** [comment]
copied to kodi's ppa for ubuntu. the other bits in our build system need some bumps, i can probably do those tomorrow while doing some libCEC work

**Pull request closed without merging**